iron-form.html

```diff
@@ -361,17 +384,19 @@
         var submittable = [];
 
         for (var i = 0; i < nodes.length; i++) {
+          var additionalElements = [];
           var node = nodes[i];
-          // An element is submittable if it is not disabled, and if it has a
```

Comment:
Comments are nice. Does it make sense to leave it in, and expand on it? Otherwise coming back to this code 6 months from now will be hard to understand

Reply:
I felt that the if statement was the same as the comment, but I will happily add it back 😄

Reply:
yeah, but one is in english and explains the next 8 lines of code, and one is in code and i have to mentally translate it to english when a bug appears :)